db/smt-db

```diff
@@ -304,7 +305,7 @@ sub setup
     my ($login,$pass,$uid,$gid) = getpwnam("wwwrun");
     chown 0, $gid, "/etc/smt.conf";
 
-    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/2.03/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
+    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/3.00/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
```

Comment:
Either latest, or sort it by version. At openQA we use DBIx which manages this by itself and it just sort it by schema version.
But I agree, this hard coded way is wrong.